src/styles/blocks/card.scss

```diff
@@ -0,0 +1,83 @@
+.card {
+  width: $card-dimension;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+
+  &__container {
+    display: flex;
+    flex-direction: column;
+    color: $description-color-text;
+  }
+
+  &__image {
+    margin-bottom: 40px;
+    align-self: center;
+  }
+
+  &__title {
+    margin-bottom: 4px;
+    font-size: 12px;
+    line-height: 18px;
+    font-weight: 500;
+  }
+
+  &__code {
+    margin-bottom: 16px;
+    font-size: 10px;
+    line-height: 14px;
+    color: $description-color;
+  }
+
+  &__reviews {
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-end;
+    margin-bottom: 24px;
+  }
+
+  &__about-stars {
+    font-size: 10px;
+    line-height: 14px;
+  }
+
+  &__price-value {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 16px;
+  }
+
+  &__price {
+    font-size: 12px;
+    line-height: 18px;
+    color: $description-color;
+  }
+
+  &__value {
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 18px;
+  }
+
+  &__button {
+    text-align: center;
+    width: 166px;
+    height: 40px;
+    border: 1px solid transparent;
+    background: $button-colors;
+    border-radius: 5px;
+    text-transform: uppercase;
+    color: #fff;
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 40px;
+
+    transition: all 0.4s ease-out;
```

Comment:
1 - let's create variable for duration (you may use it for every transition to make transitions consistent)
2 - better to define which exact properties will be animated, because if you not define it, it will be applied to every property that can be animated